aw_client/client.py

```diff
@@ -84,6 +84,11 @@ def replace_last_event(self, bucket, event: Event):
         data = event.to_json_dict()
         self.dispatch_thread.add_request(endpoint, data)
 
+    def heartbeat(self, bucket, event: Event, pulsetime: float):
+        endpoint = "buckets/{}/heartbeat?pulsetime={}".format(bucket, pulsetime)
+        data = event.to_json_dict()
+        self.dispatch_thread.add_request(endpoint, data)
```

Comment:
I think this is fine.

If one event is 500 bytes and we queue one per second then after **one week** we 7*24*60*500 = 5040000 bytes = ~5MB of queued events. It think it sounds like a microoptimization that probably adds unnecessary complexity, but one way to do it in a simple way would be to move the heartbeat merging logic (in aw-server) into aw-core and simply call it to get the job done.

I don't think it is important right now, but feel free to do it the way I proposed. :)